strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFile/Classes/ResourcesManager.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Globalization;
+using System.Reflection;
+using System.Resources;
+
+namespace ReadStringFromResourceFile;
+
+public class ResourcesManager
```

Comment:
This class is too complicated for the scope of this article. Also, the name is too similar to the built-in `ResourceManager` class. Why not use a different name such as `ResourcesUtils`?

Also, it could be a static class with only two methods, for example:

- `public string GetWithResourceManager(string key)`
- `public string GetDirectly(string key)`

Both methods could be static